index.php

```diff
@@ -32,6 +32,8 @@
 use ZipStream\Option\Archive;
 use ZipStream\ZipStream;
 
+
```

Comment:
can you please remove the 2 added lines here? - they should be triggering coding guideline failures.